views/content/crowdsale.md

```diff
@@ -8,7 +8,7 @@ Sometimes a good idea takes a lot of funds and collective effort. You could ask
 
 In this example we will make a better crowdfunding by solving two important problems: how rewards are managed and kept, and how the money is spent after the funds are raised. 
 
-Rewards in crowdfundings are usually handled by a central unchangeable database that keeps track of all donors: anyone who lost the campaign cannot get in anymore and any donor who changed their mind can't get out. Instead we are going to do this the decentralized way and just create a [token](./token) to keep track of rewards, anyone who contributes get a token that they can trade, sell or keep for later. When the time comes to give the physical reward the producer only needs to exchange the tokens for real products. Donors get to keep their tokens, even if the project doesn't achieve its goals, as a souvenir.
+Rewards in crowdfundings are usually handled by a central unchangeable database that keeps track of all donors: anyone who lost the deadline for the campaign cannot get in anymore and any donor who changed their mind can't get out. Instead we are going to do this the decentralized way and just create a [token](./token) to keep track of rewards, anyone who contributes get a token that they can trade, sell or keep for later. When the time comes to give the physical reward the producer only needs to exchange the tokens for real products. Donors get to keep their tokens, even if the project doesn't achieve its goals, as a souvenir.
```

Comment:
👍

On Wed, Mar 16, 2016 at 12:03 PM, Alexandre Van de Sande <
notifications@github.com> wrote:

> In views/content/crowdsale.md
> <https://github.com/ethereum/ethereum-org/pull/207#discussion_r56361628>:
>
> > @@ -8,7 +8,7 @@ Sometimes a good idea takes a lot of funds and collective effort. You could ask
> >
> >  In this example we will make a better crowdfunding by solving two important problems: how rewards are managed and kept, and how the money is spent after the funds are raised.
> >
> > -Rewards in crowdfundings are usually handled by a central unchangeable database that keeps track of all donors: anyone who lost the campaign cannot get in anymore and any donor who changed their mind can't get out. Instead we are going to do this the decentralized way and just create a [token](./token) to keep track of rewards, anyone who contributes get a token that they can trade, sell or keep for later. When the time comes to give the physical reward the producer only needs to exchange the tokens for real products. Donors get to keep their tokens, even if the project doesn't achieve its goals, as a souvenir.
> > +Rewards in crowdfundings are usually handled by a central unchangeable database that keeps track of all donors: anyone who lost the deadline for the campaign cannot get in anymore and any donor who changed their mind can't get out. Instead we are going to do this the decentralized way and just create a [token](./token) to keep track of rewards, anyone who contributes get a token that they can trade, sell or keep for later. When the time comes to give the physical reward the producer only needs to exchange the tokens for real products. Donors get to keep their tokens, even if the project doesn't achieve its goals, as a souvenir.
>
> Maybe "missed the deadline"?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ethereum/ethereum-org/pull/207/files/46538c8a89cc87800fffbbccbfa72d67910719ee..f151345724b61c5112abd2b8a22cd7edd1fafd9d#r56361628>
>